providers/aws/sg.go

```diff
@@ -238,9 +238,14 @@ func (g *SecurityGenerator) InitResources() error {
 	}
 	svc := ec2.New(config)
 	p := ec2.NewDescribeSecurityGroupsPaginator(svc.DescribeSecurityGroupsRequest(&ec2.DescribeSecurityGroupsInput{}))
+	var resourcesToFilter []ec2.SecurityGroup
 	for p.Next(context.Background()) {
-		g.Resources = append(g.Resources, g.createResources(p.CurrentPage().SecurityGroups)...)
+		resourcesToFilter = append(resourcesToFilter, p.CurrentPage().SecurityGroups...)
 	}
+	sort.Slice(resourcesToFilter[:], func(i, j int) bool {
+		return *resourcesToFilter[i].GroupId < *resourcesToFilter[j].GroupId
```

Comment:
I think it's need `resourcesToFilter[i].GroupId < resourcesToFilter[j].GroupId`

Reply:
I want to compare strings, not pointers